website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,446 @@
+- [Other Breaking Changes](/blog/2024/04/22/0.74-yoga-improvements-preparing-for-the-new-architecture#other-breaking-changes)
+
+<!--truncate-->
+
+## Highlights
+
+### Yoga 3.0
+
+#### New Layout Behaviors
+
+React Native 0.74 includes [Yoga 3.0](https://yogalayout.dev/blog/announcing-yoga-3.0), the newest version of our layout engine. Yoga 3.0 improves layout by making styling more predictable, and supports rendering components written for the web.
+
+React Native intentionally preserves some incorrect layout behaviors, where fixing them was found to effect a signigicant number of real-world components. Layout conformance will be able to be configured more granularly in future versions of React Native.
+
+:::warning
+
+React Native [previously flipped](https://yogalayout.dev/blog/announcing-yoga-3.0#correct-handling-of-logical-edges-in-row-reverse-containers) `left`/`right` (and `start`/`end`) edges when dealing with `margin`, `padding`, or `border`, set on a `row-reverse` container. Code which previously relied on edges being inverted may need to be updated to continue rendering correctly.
+
+<table>
+<tr>
+  <th>Style</th>
+  <th>Before</th>
+  <th>After</th>
+</tr>
+<tr>
+<td width="350">
+
+```jsx
+<View
+  style={{
+    flexDirection: 'row',
+    backgroundColor: 'red',
+    margin: 10,
+    width: 200,
+    height: 100,
+  }}>
+  <View
+    style={{
+      flexDirection: 'row-reverse',
+      backgroundColor: 'blue',
+      flex: 1,
+      marginLeft: 50,
+    }}>
+    <View
+      style={{
+        backgroundColor: 'green',
+        height: '50%',
+        flex: 1,
+        marginLeft: 50,
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+<td>
+
+![Previous layout](../static/blog/assets/0.74-row-reverse-before.png)
+
+</td>
+<td>
+
+![New layout](../static/blog/assets/0.74-row-reverse-after.png)
+
+</td>
+</tr>
+</table>
+
+:::
+
+#### Support for `align-content: 'space-evenly'`
+
+Yoga 3.0 brings support for [`alignContent: 'space-evenly'`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-content#space-evenly). `space-evenly` distributes the lines in a multi-line flex container using evenly spaced gaps, placed between line and container edges.
+
+<figure>
+  <img alt="Visual reference for alignContent behaviors" src="/blog/assets/0.74-align-content.png" width="360" />
+  <figcaption><small>Source: [World Wide Web Consortium](https://www.w3.org/TR/css-align-3/#distribution-values)</small></figcaption>
+</figure>
+
+#### Support for `position: 'static'`
+
+:::info
+`position: 'static'` is supported only in the New Architecture.
+:::
+
+Elements marked as [`position: 'static'`](https://developer.mozilla.org/en-US/docs/Web/CSS/position#static) may not be offset, and are not considered when determining the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block) of an absolutely positioned element. This allows positioning an element relative to an ancestor which is not its direct parent.
```

Comment:
This can be removed now due to some in-progress changes that make this not true since every static node is a flex item, which do form stacking contexts. It is true at the release of 0.74, but since it will change soon it is probably best to just not call it out